tfhe/benches/integer/glwe_packing_compression.rs

```diff
@@ -46,26 +48,38 @@ fn cpu_glwe_packing(c: &mut Criterion) {
         256,
         comp_param.lwe_per_glwe.0 * log_message_modulus,
     ] {
-        assert_eq!(num_bits % log_message_modulus, 0);
-        let num_blocks = num_bits / log_message_modulus;
+        assert_eq!(bit_size % log_message_modulus, 0);
+        let num_blocks = bit_size / log_message_modulus;
 
         let ct = cks.encrypt_radix(0_u32, num_blocks);
 
         let mut builder = CompressedCiphertextListBuilder::new();
 
         builder.push(ct);
 
-        bench_group.bench_function(format!("pack_u{num_bits}"), |b| {
+        let bench_id = format!("{bench_name}::pack_u{bit_size}");
+        bench_group.bench_function(&bench_id, |b| {
             b.iter(|| {
                 let compressed = builder.build(&compression_key);
 
                 _ = black_box(compressed);
             })
         });
 
+        write_to_json::<u64, _>(
+            &bench_id,
+            comp_param,
+            "COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64",
+            "unpack",
```

Comment:
pack or unpack ?

Reply:
yes but I think this result is for the pack not unpack ?

Reply:
You're right.